app/controllers/registration_command_controller.rb

```diff
@@ -0,0 +1,28 @@
+class RegistrationCommandController < ApplicationController
```

Comment:
Maybe I've missed discussion, so please excuse me. Why renaming is needed? Even if so, why not `GlobalRegistration` or even `HostRegistrationCommand`? Not a blocker tho, just thoughts so you don't have to rename it in Katello :D

Reply:
> Why renaming is needed? 

It's not _needed_, I just wanted to have it same way as in `api` scope:
`RegistrationController` - for the host registration itself
`RegistrationCommand` - for the command stuff only

With this one would expect that `UI` controllers would be same, right?
But no strong opinions here, just wanted to have it same in both scopes. 

> Why renaming is needed? Even if so, why not GlobalRegistration or even HostRegistrationCommand?

Well, `RegistrationController` is shorter than `GlobalRegistrationController` or `HostRegistrationCommandController`,
plus we still don't have official branding name for this feature, so `Registration` was nice compromise.